torch/_tensor_str.py

```diff
@@ -275,11 +275,13 @@ def get_summarized_data(self):
     else:
         return torch.stack([get_summarized_data(x) for x in self])
 
-def _str_intern(self):
+def _str_intern(inp):
     prefix = 'tensor('
     indent = len(prefix)
     suffixes = []
 
+    self, tangent = torch.autograd.forward_ad.unpack_dual(inp)
```

Comment:
If I unpack a multi-level dual tensor, self could still be dual. So this code doesn't work in the multi level case, and needs a TODO comment saying that it needs to be fixed later.